libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -89,17 +89,17 @@
         />
         <span
             class="fd-input-group__addon fd-input-group__addon--button"
+            [ngClass]="addOnClass"
             [class.fd-input-group__addon--compact]="_contentDensityObserver.isCompact$ | async"
-            [class.fd-shellbar__input-group-addon]="inShellbar"
-            *ngIf="isSearch && showClearButton && inputText && inputText.length > 0"
+            [style.display]="isSearch && showClearButton && inputText && inputText.length > 0 ? '' : 'none'"
```

Comment:
ternary here resolves to `''`, what this stands for?

Reply:
@shrvr ?

Reply:
Hm, it's to set display to `none` only when condition is false. `''` won't add display property at all in case of true condition. Otherwise, I can add `'block'` instead `''`. @platon-rov @InnaAtanasova